tests/data/bundles/test_core.py

```diff
@@ -253,6 +264,66 @@ def bundle_ingest(environ,
             msg='volume',
         )
 
+    def test_ingest_assets_versions(self):
+        versions = (1, 2)
+
+        called = [False]
+
+        @self.register('bundle', create_writers=False)
+        def bundle_ingest_no_create_writers(*args, **kwargs):
+            called[0] = True
+
+        with self.assertRaises(ValueError):
```

Comment:
wgy does the first ingest fail? Can we use `assert_raises_regex` to make this more clear?